client/app/components/Modal/index.js

```diff
@@ -0,0 +1,152 @@
+import React from 'react';
+import ReactModal from 'react-modal';
+import styles from "./styles.scss";
+import bulma from 'bulma';
+import avatar from "./IDICIA-logo-clear-center.svg";
+
+class ContactModal extends React.Component {
+    constructor () {
+      super();
+      this.state = {
+        showModal: false
+      };
+      
+      this.handleOpenModal = this.handleOpenModal.bind(this);
+      this.handleCloseModal = this.handleCloseModal.bind(this);
+    }
+    
+    handleOpenModal () {
+      this.setState({ showModal: true });
+    }
+    
+    handleCloseModal () {
+      this.setState({ showModal: false });
+    }
+    
+    render () {
+      return (
+        <div className="navbar-item animated fadeInUp">
+          <button className="button is-link" onClick={this.handleOpenModal}>Contact Us</button>
+          <ReactModal 
+             isOpen={this.state.showModal}
+             contentLabel="Minimal Modal Example"
+             style={{
+                overlay: {
+                  backgroundColor: 'rgba(221, 221, 221, 0.9)',
+                },
+                content: {
+                    border                     : '2px solid #fff',
+                    background                 : '#dddddd',
+                    borderRadius               : "20px"
+                 
+                }
+              }}
+          >
+            {/* avatar for brand image */}
+                {/* <div className="column is-one-fifth">
+                </div> */}
+                    <figure>
+                            <img className={`${styles.img}`} src={avatar} />
+                    </figure>                
+            {/* bulma form */}
+            <div className="field is-horizontal">
```

Comment:
@leseanbiz If this form is expected to work at somepoint... it needs to actually be a `<form>` with submission buttons and behavior. Semantically, this isn't a form yet.